### PR TITLE
add how to dump to amazon s3 for dumpling

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)
add how to dump to amazon s3 for dumpling

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -80,6 +81,34 @@ dumpling \
 >
 > 2. 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
 
+### 导出到 Amazon S3 云盘
+
+Dumpling 在 v4.0.8 版本及更新版本中支持导出到云盘。如果需要将数据备份到 Amazon 的 S3 后端存储，那么需要在 `-o` 参数中指定 S3 的存储路径。
+
+这里可以参照 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-bucket.html)在指定的 `Region` 区域中创建一个 S3 桶 `Bucket`，如果有需要，还可以参照 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-folder.html) 在 Bucket 中创建一个文件夹 `Folder`。
+
+将有权限访问该 S3 后端存储的账号的 `SecretKey` 和 `AccessKey` 作为环境变量传入 Dumpling 节点。
```
Comment (kennytm) [inline dumpling-overview.md:90]: we also support reading the `[default]` credentials from [the `~/.aws/credentials` file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html). perhaps link to the "S3 参数" section in `br/backup-and-restore-storages.md`.

**Review by kennytm — COMMENTED** [review]
Rest LGTM

File: dumpling-overview.md (hunk 1 above)
Comment (lichunzhu) [reply dumpling-overview.md:90]: added in https://github.com/pingcap/docs-cn/pull/4626/commits/2e1c86027662a792d36694b9f5f9402889a29fcb

File: dumpling-overview.md [hunk 2]
```diff
@@ -80,6 +81,36 @@ dumpling \
...
+{{< copyable "shell-regular" >}}
+
+```shell
+export AWS_ACCESS_KEY_ID=${AccessKey}
+export AWS_SECRET_ACCESS_KEY=${SecretKey}
+```
+
+Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。更多 Dumpling 存储配置可以参考与之一致的 [BR 存储](/br/backup-and-restore-storages.md)。
+
+在进行 Dumpling 备份时，显示指定参数 `--s3.region`，表示 S3 存储所在的区域。
```
Comment (TomShawn) [inline dumpling-overview.md:101]:
```suggestion
在进行 Dumpling 备份时，显式指定参数 `--s3.region`，即表示 S3 存储所在的区域。
```

File: dumpling-overview.md [hunk 3]
```diff
@@ -80,6 +81,36 @@ dumpling \
 >
 > 2. 这里需要在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
 
+### 导出到 Amazon S3 云盘
+
+Dumpling 在 v4.0.8 版本及更新版本中支持导出到云盘。如果需要将数据备份到 Amazon 的 S3 后端存储，那么需要在 `-o` 参数中指定 S3 的存储路径。
```
[the remark this answers is not in the document]
Comment (lichunzhu) [reply dumpling-overview.md:86]: `nightly` version has this feature. Maybe we can merge this PR later.

File: dumpling-overview.md [hunk 4]
```diff
@@ -80,6 +81,36 @@ dumpling \
...
+
+可以参照 [AWS 官方文档 - 如何创建 S3 存储桶](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-bucket.html)在指定的 `Region` 区域中创建一个 S3 桶 `Bucket`。如果有需要，还可以参照 [AWS 官方文档 - 创建文件夹](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-folder.html) 在 Bucket 中创建一个文件夹 `Folder`。
```
Comment (tennix) [inline dumpling-overview.md:88]: We'd better use the English version link and let users change the language when they are viewing the aws document.

Comment (TomShawn) [reply dumpling-overview.md:88]: We will add the English version link in the English version of this document. Usually, the language of external pages should be consistent with our document language. If the external page doesn't have the corresponding Chinese one, we should specify the language in our doc. For example, [AWS 官方文档 - 创建文件夹（英语）](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/user-guide/create-folder.html).

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**